src/service-worker-registration.ts

```diff
@@ -77,22 +76,22 @@ function registerValidSW (swUrl: string, config?: Config) {
         installingWorker.onstatechange = () => {
           if (installingWorker.state === 'installed') {
             if (navigator.serviceWorker.controller) {
-            // At this point, the updated precached content has been fetched,
-            // but the previous service worker will still serve the older
-            // content until all client tabs are closed.
+              // At this point, the updated precached content has been fetched,
+              // but the previous service worker will still serve the older
+              // content until all client tabs are closed.
               console.log(
                 'New content is available and will be used when all ' +
-              'tabs for this page are closed. See https://cra.link/PWA.'
+                'tabs for this page are closed. See https://cra.link/PWA.'
               )
```

Comment:
Customize or remove this message?
(For me it's not about the message but about the cra.link which could instead point to our FAQ)

Reply:
I removed this message, because we only need the callback.